service/src/main/resources/api/service_openapi.yaml

```diff
@@ -867,6 +867,36 @@ paths:
         500:
           $ref: "#/components/responses/ServerError"
 
+  "/v2/studies/{studyId}/conceptSets/{conceptSetId}/clone":
```

Comment:
Created a new path (this) instead of adding POST to /v2/studies/{studyId}/conceptSets/{conceptSetId} 
This was to avoid confusion with PATCH /v2/studies/{studyId}/conceptSets/{conceptSetId} used for update

Reply:
From discussion: we will keep the new {id}/clone path, similar to other Workbench services